plugin.video.filmarkivet/tests/filmarkivet_unit.py

```diff
@@ -0,0 +1,185 @@
+import unittest
```

Comment:
Please don't include this file. It seems to be for testing only?

Reply:
Yes, that's true, I have removed the file now.

Thank you for the review!